sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -399,6 +399,13 @@ public class SentryOptions {
   private @NotNull TransactionPerformanceCollector transactionPerformanceCollector =
       NoOpTransactionPerformanceCollector.getInstance();
 
+  /** Enables the time-to-full-display spans in automatic ui transactions. */
```

Comment:
```suggestion
  /** Enables the time-to-full-display spans in activity transactions. */
```
there are multiple occurences of "automatic ui transactions" in this file, so I'd also fix the wording there. (I believe we can omit the "automatic" part as well, because "activity transaction" is already self-descriptive, we always assume they are automatic when talking about them)

Reply:
The thing is that this is Java, not Android SDK. Is it fine to talk explicitly about Activities?
What about `ui transactions` without automatic?